pkg/apis/cluster/v1alpha1/cluster_types.go

```diff
@@ -92,6 +98,9 @@ const (
 	// Cluster has been one of federated clusters
 	ClusterFederated ClusterConditionType = "Federated"
 
+	// Cluster external ready
+	ClusterExternalReady ClusterConditionType = "ExternalReady"
```

Comment:
`ClusterExternalReady` is ambiguous,  how about `ClusterExternalAccessReady`

Reply:
changed